core/services/job/orm.go

```diff
@@ -468,7 +468,7 @@ func (o *orm) CreateJob(jb *Job, qopts ...pg.QOpt) error {
 
 // ValidateKeyStoreMatch confirms that the key has a valid match in the keystore
 func ValidateKeyStoreMatch(spec *OCR2OracleSpec, keyStore keystore.Master, key string) error {
-	if spec.PluginType == types.Mercury {
+	if spec.PluginType == types.Mercury || spec.PluginType == types.LLO {
```

Comment:
A switch could be a little cleaner:
```suggestion
	switch spec.PluginType {
	case  types.Mercury, types.LLO:
```